MediaPlayer.py

```diff
@@ -0,0 +1,55 @@
+import re
+from pygame import mixer
+import os
+
+
+# p = "C:\Users\Mulat\Desktop\MediaPlayerProject\songs"
+
+
+def load_song_dir_to_dictionary() -> dict:
+    user_path = input()
```

Comment:
I think the user interface shouldn't be controlled by this function, so maybe it should take this variable as an input. In general - try to separate user interface and functionality.